guides/release/accessibility/learning-resources.md

```diff
@@ -1,31 +1,32 @@
-These accessibility learning resources will provide additional support to the developer looking to improve the quality of the code they write. 
+Ces ressources d'apprentissage sur l'accessibilité ont pour but d'apporter une aide additionnelle aux développeurs.euses pour améliorer la qualité de leur code.
 
-- [MDN Web Docs: Accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility)
-- [Using ARIA:](https://www.w3.org/TR/using-aria/) a practical guide for developers on how to add accessibility information to HTML elements
-- [Web Content Accessibility Guidelines(WCAG) 2.1](https://www.w3.org/TR/WCAG21/)
-- [Accessible Rich Internet Applications (ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/) 
-- [How to meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG21/quickref/)
+*Note: Les liens ci-dessous redirigent vers des articles écrits en Anglais
 
+- [Documentation Web MDN: Accessibilité](https://developer.mozilla.org/en-US/docs/Learn/Accessibility)
+- [Utiliser ARIA:](https://www.w3.org/TR/using-aria/) un guide pratique pour développeurs.euses: Comment ajouter des informations d'accessibilité sur des éléments HTML
+- [Web Content Accessibility Guidelines(WCAG): Directives du contenu web sur l'accessibilité 2.1](https://www.w3.org/TR/WCAG21/)
+- [Accessible Rich Internet Applications (ARIA): Applications internet riches et accessibles 1.1](https://www.w3.org/TR/wai-aria-1.1/)
+- [Comment répondre aux directives WCAG (référence rapide)](https://www.w3.org/WAI/WCAG21/quickref/)
 
-## Practical Resources
+## Ressources pratiques
 
-- [Accessibility Insights](https://accessibilityinsights.io/)- Use FastPass to find common, high-impact issues 
-- [aXe extension for Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)
-- [Accessibility Support](https://a11ysupport.io/)- Find out if your code will work with assistive tech
-- [How and where to report accessibility bugs](https://www.digitala11y.com/how-where-to-report-accessibility-bugs/)
+- [Aperçus sur l'accessibilité](https://accessibilityinsights.io/)- Utiliser FastPass pour trouver des problèmes communs à fort impacts
+- [extension aXe pour Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)
+- [Aide sur l'accessibilité](https://a11ysupport.io/)- Comprendre si son code marche avec les technologies d'assistance
+- [Où et comment ajouter un bug d'accessibilité](https://www.digitala11y.com/how-where-to-report-accessibility-bugs/)
 
 
-### Design 
+### Design
 
-- [ColorSafe](http://colorsafe.co/)- Empowering designers with beautiful and accessible color palettes based on WCAG Guidelines of text and background contrast ratios
-- [Accessible Color Palette Builder](https://toolness.github.io/accessible-color-matrix/)
+- [ColorSafe](http://colorsafe.co/)- Aider les designers avec de magnifiques palettes de couleurs accessibles, basées sur les directives WCAG, ratio de couleurs de contraste de textes et arrières plans.
```

Comment:
I suggest "Appuyer" which corresponds maybe a bit more to "Empowering" while "Aider" is more about solving an issue they couldn't solve themselves. The sentence makes a bit more sense with "avec ratio ..." so I would also replace the first occurrence to avoid repetition.

```suggestion
- [ColorSafe](http://colorsafe.co/)- Appuyer les designers en proposant de magnifiques palettes de couleurs accessibles, basées sur les directives WCAG, avec ratio de couleurs de contraste de textes et arrières plans.
```